src/bin/gql-gen.ts

```diff
@@ -53,6 +51,7 @@ function libDir(cwd: string): string {
  * Generate TypeScript typings for GraphQL schema entities.
  */
 async function main(): Promise<void> {
+    configureLogging(NoLoggingConfiguration);
```

Comment:
Why wouldn't `NoLoggingConfiguration` just be the default and then we can omit all this nonsense?

Reply:
Do you think the straight to console.log is a good default? I think it probably is.